templates/JSResolverOCHTTPSTemplate.js

```diff
@@ -940,7 +936,6 @@ function refactorGremlinqueryOutput(queryResult, fieldsAlias) {
     let data = '';
     let isScalar = false;
     let isOneElement = false;
-    let isArray = false;
```

Comment:
Isn't `isArray` still referenced by line 944?
```
isArray = true;
```

Reply:
whoops yeah it is. i'll add it back